### ISSUE TITLE
Yes, but what does reusable mean?

### ISSUE THREAD

**Issue by nycdotnet** [dotnet/docs#5150]
The main thing this document lacks is some examples of circumstances in which a handler could be reusable or not.  Like, for example, does having member variables prevent re-usability?  Unmanaged references?  Why would I want to make something reusable?  Assuming performance?  Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 75c6cb21-c95c-f948-878b-f908e1449512
* Version Independent ID: ae1bd215-03e3-7a44-783a-9c7189432aa4
* Content: [IHttpHandler.IsReusable Property (System.Web)](https://docs.microsoft.com/en-us/dotnet/api/system.web.ihttphandler.isreusable?view=netframework-4.7.1#System_Web_IHttpHandler_IsReusable)
* Content Source: [xml/System.Web/IHttpHandler.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Web/IHttpHandler.xml)
* Service: **unspecified**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
@Rick-Anderson @scottaddie this is feedback for an ASP.NET API. Do you know someone who can help with these questions?

**Comment by Rick-Anderson** [comment]
@danroth27  what  makes a handler reusable?

**Comment by danroth27** [comment]
> @danroth27 what makes a [`IHttpHandler`] reusable?

@Eilon @DamianEdwards Can you guys answer this one?

**Comment by Rick-Anderson** [comment]
@nycdotnet  [Stack Overflow](https://www.google.com/search?ei=u67rWqLwDs62jwO84KKgAQ&q=IHttpHandler.IsReusable+stackoverflow&oq=IHttpHandler.IsReusable+stackoverflow&gs_l=psy-ab.3...57606.63851.0.64146.20.20.0.0.0.0.219.2239.0j16j1.17.0....0...1c.1.64.psy-ab..3.12.1545...0i30k1j0i5i30k1j33i22i29i30k1j33i160k1j33i13i21k1.0.96Y6sTQ7x4k) seems to have good answers.

**Comment by nycdotnet** [comment]
Agreed - the answer from Branislav sounds pretty good.  If it's right, would be wonderful to get the constraints in the docs.  Thank you!
